packages/flutter_markdown/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
We don't "pub publish" any of the code in the "packages" directory.  Instead, developers get the code by cloning the flutter.git repository.